mlflow/pytorch/__init__.py

```diff
@@ -46,8 +46,43 @@
 
 def get_default_conda_env():
     """
-    :return: The default Conda environment for MLflow Models produced by calls to
+    :return: The default Conda environment as a dictionary for MLflow Models produced by calls to
              :func:`save_model()` and :func:`log_model()`.
+
+    .. code-block:: python
+        :caption: Example
+
+        import numpy as np
+        import torch
+        import mlflow.pytorch
+
+        # Class defined here
+        class LinearNNModel(torch.nn.Module):
+            ...
+
+        # Initialize our model, criterion and optimizer
+        model = LinearNNModel()
+        ...
+
+        # Training loop
+        ...
+
+        # Log PyTorch model
+        with mlflow.start_run() as run:
+            mlflow.pytorch.log_model(model, "models_pth")
```

Comment:
```suggestion
        import mlflow.pytorch
```

Can we remove unrelated code?

Reply:
You're minimalist, aren't you? :-) 

Why do you feel that this code is unrelated? This provides some context, so If they want to try a full example, then can
use look at the full example above. At the least, we should have:

        # Class defined here
        class LinearNNModel(torch.nn.Module):
            ...
        # Initialize our model, criterion and optimizer
        ...

        # Training loop
        ...

Reply:
If we had:

```
        # Class defined here
        class LinearNNModel(torch.nn.Module):
            ...
        # Initialize our model, criterion and optimizer
        ...

        # Training loop
        ...
```

it would look like the class definition and training loop affect the result of `get_deafult_conda_env`.

Reply:
Ah, I see what you mean! 

(The reason I had it is, because if you had just trained and logged a PyTorch model in the elided code above, then you can fetch its conda environment with this API code.)

I can remove it.